dataverse-webapp/src/test/java/edu/harvard/iq/dataverse/export/SchemaDotOrgExporterTest.java

```diff
@@ -257,8 +256,9 @@ public void testExportDataset() throws Exception {
         dataverse.setName("LibraScholar");
         dataset.setOwner(dataverse);
         boolean hideFileUrls = false;
+        String hideFileUrlsString="false";
         if (hideFileUrls) {
```

Comment:
Why is this if needed? The value will always be false.

Reply:
I guess it was added to be able to steer test behavior, but i might delete it now aswell